FhirToDataLake/scripts/Set-SynapseEnvironment.ps1

```diff
@@ -376,34 +376,58 @@ function Get-OrasExeApp {
     Write-Host "Finish download oras application from $orasUrl" -ForegroundColor Green 
 }
 
+function Get-ImageDigest {
+    param ([string]$orasAppPath, [string]$schemaImageReference)
```

Comment:
Shall we add some logging information here?

Reply:
Sure, added a log inside this method.